src/equisolve/numpy/utils.py

```diff
@@ -32,3 +35,16 @@ def block_to_array(block: TensorBlock, parameter_keys: List[str]):
             data = block.gradient(parameter).data
             M.append(data.reshape(np.prod(data.shape[:-1]), data.shape[-1]))
     return np.vstack(M)
+
+def tensor_map_to_dict(tensor_map: TensorMap):
```

Comment:
There should be a comment in there to remove this function once https://github.com/lab-cosmo/equistore/issues/94 is implemented

Reply:
added